loopdb/sqlite.go

```diff
@@ -377,12 +378,25 @@ func parseTimeParts(datePart, timePart string) (time.Time, error) {
 		return time.Time{}, err
 	}
 
-	day, err := strconv.Atoi(dateParts[2])
+	dateIndex := 2
```

Comment:
I find it hard to parse what the code here is doing. Maybe we can add the template date string that is handled here to clarify?

Reply:
Also, maybe in the two returns above maybe we can add `unable to atoi month` and `unable to atoi year` to the error string.